scripts/kind-e2e/e2e.sh

```diff
@@ -183,39 +177,36 @@ function test_connection() {
 
 function update_subm_pods() {
     echo Removing submariner engine pods...
-    kubectl delete pods -n submariner -l app=submariner-engine
-    kubectl wait --for=condition=Ready pods -l app=submariner-engine -n submariner --timeout=60s
+    kubectl --cluster=$1 delete pods -n submariner -l app=submariner-engine
```

Comment:
--context= ? 

Isn't this overlapping with the other PR ?

Reply:
Both `--cluster` and `--context` work, but I’ve updated the PR to only use `--context`. #110 includes this PR (see the comment in the description of #110).